website/content/docs/upgrading/upgrade-to-1.15.x.mdx

```diff
@@ -40,3 +40,11 @@ To continue measuring `vault.rollback.attempts.{MOUNTPOINT}` and
 metrics in the `telemetry` stanza of your Vault configuration with the
 [`add_mount_point_rollback_metrics`](/vault/docs/configuration/telemetry#add_mount_point_rollback_metrics)
 option.
+
+## Application of Sentinel Role Governing Policies (RGPs) via identity groups
```

Comment:
If you add this content in `website/content/partials`, you'll be able to make it a partial and import it into this upgrade guide and the 1.13 and 1.14 upgrade guides more easily.

Reply:
That's very useful. Thanks, Meggie!